### PR TITLE
Feature/documentation

### PR DESCRIPTION
**Pull request by BeatriceDuncan**
### Notion card link(s)

- [Documentation](https://www.notion.so/normative/Documentation-8ba03e76bf9446d1a77873d8eae95e1f)

### Description (Proposed changes, additional context, chosen approach/alternative approaches, gotchas, etc)

- Is the Data / Glossary section necessary/helpful? 
- Analytics section to be updated once we have a clear analytics strategy moving forward.
- Could add a link out to the Retool documentation once it's completed.

### Steps for reviewer(s) to reproduce/test

- Check live links
- Is there anything I've missed / don't need here?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,20 +2,68 @@
 
 `yarn` (install dependencies)
 
-`yarn start`
+`cp .env.example .env` (create .env config)
```
Comment (tonybennett) [inline README.md]: Wondering if we should remove this .env.example, or some of the values knowing now that it's a public repo (not that we have secrets in there, but thinking of best practices). We already have the instructions that they must get the creds from heroku which is the important part

File: README.md [hunk 2]
```diff
@@ -2,20 +2,68 @@
...
 Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.<br>
+
+localhost:3000 automatically re-directs to the [dtpr homepage](https://dtpr.helpfulplaces.com/), provide url paramaters for either sensorId or placeId to view/test app:<br>
+E.g. `{baseUrl}/sensors/{sensorId}` or `{baseUrl}/sensors/{sensorId}/print` or `{baseUrl}/place/{placeId}`<br>
+Quick start:<br>
+Place: [Noramtive](http://localhost:3000/places/Ur-FM0fJw)<br>
+Sensor: [Demo sensor](http://localhost:3000/sensors/1iAQWQyJp50uWc8ACxJZ)<br>
+SensorPrint: [Demo sensor](http://localhost:3000/sensors/1iAQWQyJp50uWc8ACxJZ/print)<br>
```
Comment (tonybennett) [inline README.md:18]: Like the example, although it's likely tied to one region? Or did we propagate these across both regions?

**Review by tonybennett — COMMENTED** [review]
Looks good and useful, this feels like the right info for a developer

File: README.md (hunk 2 above)
Comment (BeatriceDuncan) [reply README.md:18]: Correct it _is_ tied to one region. Thanks, I will be clear about this. (Or I can remove if preferred.)

**Review by turnKei — APPROVED** [review]

**Pull request merged**